conductr_cli/conduct_agents.py

```diff
@@ -38,12 +44,25 @@ def agents(args):
                 'roles': ','.join(entry['roles']),
                 'observed': ','.join(map(lambda e: e['node']['address'], entry['observedBy']))
             })
+            if with_resources:
+                data[-1]['disk-space'] = naturalsize(entry['resourceAvailable']['diskSpace'])
+                data[-1]['memory'] = naturalsize(entry['resourceAvailable']['memory'], binary=True)
+                data[-1]['nr-of-cpus'] = entry['resourceAvailable']['nrOfCpus']
 
     padding = 2
     column_widths = dict(screen_utils.calc_column_widths(data), **{'padding': ' ' * padding})
 
     for row in data:
-        log.screen('''\
+        if with_resources:
+            log.screen('''\
+{address: <{address_width}}{padding}\
+{disk-space: <{disk-space_width}}{padding}\
+{memory: <{memory_width}}{padding}\
+{nr-of-cpus: <{nr-of-cpus_width}}{padding}\
```

Comment:
Maybe the `disk-space`, `memory`, and `nr-of-cpus` should be right aligned - use `>` instead of `<`, i.e.

```
{disk-space: >{disk-space_width}}{padding}\
{memory: >{memory_width}}{padding}\
{nr-of-cpus: >{nr-of-cpus_width}}{padding}\
```